laboratory/serializers.py

```
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+from .models import LabResult
+
+
+class LabResultSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = LabResult
+        fields = ['id', 'patient', 'encounter', 'loinc', 'value', 'unit', 'taken_at']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)